src/theme/Theme.js

```diff
@@ -9,8 +9,27 @@ const theme = {
     grayLight: '#e6e6e6',
     grayMedium: '#93918f',
     grayDark: '#636363',
+    grayDarker: '#787878',
     buttonBg: '#fdb755',
     link: '#0087ff',
+    transparent: 'transparent',
+    backgroundDark: '#1e2537',
+    borderDark: '#1e2537',
+    backgroundBottom: '#e9e9e9',
+    backgroundTop: '#fefefe',
+  },
+  heatMapColors: {
```

Comment:
It might be a good idea to have a heatmap object here and include more constants like the cell width and height, the weekday width and so on. The colors can be included there as well. 

It might make sense to change the data structure of the colors here as well. Instead of using `color*` as key you can simply use a number `0: ...`. Then you don't need to create keys in the styled component like this

```
props.theme.heatMapColors[`color${props.children}`]
```

But when we're already at it you can also use an array. If you want to be more consistent and only use objects in the theme it makes sense as well